x/operator/client/cli/query.go

```diff
@@ -225,3 +228,93 @@ func GetAllOperatorConsAddrs() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// QueryOperatorUSDValue queries the opted-in USD value for the operator
+func QueryOperatorUSDValue() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "QueryOperatorUSDValue operatorAddr avsAddr",
+		Short: "Get the opted-in USD value for the operator",
+		Long:  "Get the opted-in USD value for the operator",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := operatortypes.NewQueryClient(clientCtx)
+			req := &operatortypes.QueryOperatorUSDValueRequest{
+				Details: &operatortypes.OperatorAVSAddressDetails{
+					OperatorAddr: args[0],
+					AVSAddress:   args[1],
+				},
+			}
+			res, err := queryClient.QueryOperatorUSDValue(context.Background(), req)
+			if err != nil {
+				return err
+			}
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+	return cmd
+}
+
+// QueryAVSUSDValue queries the USD value for the avs
+func QueryAVSUSDValue() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "QueryAVSUSDValue avsAddr",
+		Short: "Get the USD value for the avs",
+		Long:  "Get the USD value for the avs",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := operatortypes.NewQueryClient(clientCtx)
+			req := &operatortypes.QueryAVSUSDValueRequest{
+				AVSAddress: args[0],
```

Comment:
Please check that it is hex

Reply:
The AVS address of the dog food isn't in hex. It seems that it's not necessary to check the input here. The RPC will return an error for the invalid input.

Reply:
The error returned is that the key does not exist; it does not indicate whether it's because there's no such AVS or because there's a problem in the input. It is also better to not open the store each time for an invalid query. This is called stateless validation in Cosmos. For example, `genState.Validate()` function ensures that the input is accurate without looking at state and then `keeper.InitGenesis` can process the previously validated input, performing only the stateful validaitons.

Reply:
Same comment as above.